core/src/main/java/io/parsingdata/metal/data/callback/Callback.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2013-2016 Netherlands Forensic Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.parsingdata.metal.data.callback;
+
+import io.parsingdata.metal.data.ParseResult;
+import io.parsingdata.metal.token.Token;
+
+public interface Callback {
+
+    void handle(final Token token, final ParseResult result);
```

Comment:
I've added #9 to the 5.0.0 milestone. It's better to document the important stuff first I think.